controllers/certmanager/certmanager.go

```diff
@@ -51,6 +63,15 @@ func DeployCR(bs *bootstrap.Bootstrap) {
 	}
 }
 
+// getCertSubscription return true if Cert Manager subscription found, otherwise return false
+func getCertSubscription(r client.Reader, MasterNs string) bool {
+	subName := "ibm-cert-manager-operator"
+	subNs := MasterNs
```

Comment:
Could it cover the multiple instance deployment when the cert-manager is in control namespace?

Reply:
Sure, updated!